config/WordPress.alpha.xcconfig

```diff
@@ -1,4 +1,4 @@
-#include "Version.internal.xcconfig"
+#include "Version.public.xcconfig"
```

Comment:
We weren't really using the unique internal version xcconfig build number values once the release cycle builds were disabled in March. This change will just have the alpha/prototype builds use the release version number from the public file and will still continue to generate build numbers on-demand.

Reply:
Sounds good.